docs/reference/pip_download.rst

```diff
@@ -42,10 +55,66 @@ Options
 Examples
 ********
 
-#. Download a package and all of its dependencies
-
-    ::
-
-      $ pip download SomePackage
-      $ pip download -d . SomePackage  # equivalent to above
-      $ pip download --no-index --find-links=/tmp/wheelhouse -d /tmp/otherwheelhouse SomePackage
+1. Download a package and all of its dependencies
+
+  ::
+
+    $ pip download SomePackage
+    $ pip download -d . SomePackage  # equivalent to above
+    $ pip download --no-index --find-links=/tmp/wheelhouse -d /tmp/otherwheelhouse SomePackage
+
+2. Download a package and all of its dependencies with OSX specific interpreter constraints.
+   This forces OSX 10.10 or lower compatibility. Since OSX deps are forward compatible,
+   this will also match ``macosx-10_9_x86_64``, ``macosx-10_8_x86_64``, ``macosx-10_8_intel``,
+   etc.
+   It will also match deps with platform ``any``.  Also force the interpreter version to ``27``
+   (or more generic, i.e. ``2``) and implementation to ``cp`` (or more generic, i.e. ``py``).
+
+  ::
+
+    $ pip download \
+        --only-binary=:all: \
```

Comment:
Redundant in the sense that it should now just be `--only-binary`, or that it's not needed at all?